app/code/community/Yotpo/Yotpo/controllers/Review/ExportController.php

```diff
@@ -8,13 +8,27 @@ public function csvexportAction()
     {
         $reviews = $this->getRequest()->getPost('reviews', array());
         $file = Mage::getModel('Yotpo_Yotpo_Model_Export_Csv')->exportReviews($reviews);
-        $data = Mage::getModel('Yotpo_Yotpo_Model_Export_Csv')->exportData($reviews);
-        $this->_prepareDownloadResponse($file, array('type' => 'filename', 'value' => $data));
+        //$data = Mage::getModel('Yotpo_Yotpo_Model_Export_Csv')->exportData($reviews);
+        //$this->_prepareDownloadResponse($file, array('type' => 'filename', 'value' => $data));
+        if ($file)
+            $this->_redirect('*/*/fileDownload', array('csv'=> $file));
+    }
+    
+    protected function fileDownloadAction()
+    {
+        $fileName = $this->getRequest()->getParam('csv',false);
+        $baseDir = Mage::getBaseDir("export")."/";
+        $file_content = file_get_contents($baseDir . $fileName);
+        if ($file_content) {
+            $this->getResponse()->setHeader('Content-type', 'application/csv');
+            $this->getResponse()->setBody($file_content);

```

Comment:
Needs this too: `$this->getResponse()->setHeader('Content-Disposition',` 'attachment; filename="' . $fileName . '"');`